rosbag2_performance/rosbag2_performance_benchmarking/src/benchmark_publishers.cpp

```diff
@@ -109,20 +117,34 @@ class BenchmarkPublishers : public rclcpp::Node
     producer->period = std::chrono::milliseconds(
       producer_config.frequency ? 1000 / producer_config.frequency : 1);
 
+    thread_pool_.queue(
```

Comment:
It would be nice to check that `producer->max_messages > 0` before adding producer_job to the thread pool queue.

Reply:
done